app/jobs/hyrax/valkyrie_create_derivatives_job_decorator.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+# OVERRIDE to log and report derivative errors to Sentry
```

Comment:
For consistency with other override comments, can you please include what version of Hyrax this is overriding? [Example](https://github.com/samvera/hyku/blob/32dbeaa73d0dc05365838dcdac538df7b646415d/app/models/content_block.rb#L3)